Kitodo/src/main/resources/messages/errors_en.properties

```diff
@@ -86,6 +87,21 @@ invalidIdentifierSame=The identifier {0} has the same value in {1} and {2}.
 # J
 
 # K
+kitodoScript.importProcesses.brokenRelatedProcess=Process wasn''t imported due to errors in related process(es): {0}
```

Comment:
I think we shouldn't abbreviate "not" in text that appears in the UI. Could you replace "wasn't" with "was not" and "isn't" with "is not" in the new messages?